packages/x-charts/src/material/index.ts

```diff
@@ -0,0 +1,21 @@
+import Tooltip from '@mui/material/Tooltip';
```

Comment:
Why do we have this src/material/ folder? 🤔

Reply:
It provides the default slots for the toolbar, which use Material UI

Reply:
I still don't understand. Why not have these under the `Toolbar` folders? 🤔

Reply:
For two reasons:
1. Similarity to Data Grid, which structures the code in the same way;
2. If we want to use these components outside the toolbar, it wouldn't make as much sense if they're inside the `Toolbar/` directory.

I don't feel too strongly about these reasons, though.

Reply:
I'm not too thrilled to have an export named `@mui/x-charts/material` in our current state. 

I assume this is to isolate material imports? So it is easier to change or import other stuff in the future?

Reply:
Correct. It makes it clear that these components are Material UI components. What don't you like about the export?

Reply:
Because it is useless right now, and I don't see changes to it in the near future. 😆 

If we go the "unstyled" way. We would likely provide a `@mui/x-charts/material`, as it would allow our users to import the material-styled components if they need/have the `@material` lib installed.

But this is used internally, which defeats the purpose, so I would probably just move these files to were they are actually used, and if we need a common place eventually we can have it.

This would also leave the `@mui/x-charts/material` import path free to be used as we please

Reply:
What about moving this to `src/internal/material` such that we get both

1. a clear folder where we can import material stuff
2. we don't expose stuff that user don't already need to import to stay free of any import path in the future

Reply:
Moved pro slots to the pro package 👍